cachito/workers/pkg_manager.py

```diff
@@ -100,25 +100,27 @@ def update_request_with_deps(request_id, deps, env_vars=None):
     request_url = f'{config.cachito_api_url.rstrip("/")}/requests/{request_id}'
 
     log.info('Adding %d dependencies to request %d', len(deps), request_id)
-    payload = {'dependencies': deps}
-    if env_vars:
-        log.info('Adding environment variables to request %d: %s', request_id, env_vars)
-        payload['environment_variables'] = env_vars
-    try:
-        rv = requests_auth_session.patch(
-            request_url, json=payload, timeout=config.cachito_api_timeout)
-    except requests.RequestException:
-        msg = f'The connection failed when setting the dependencies on request {request_id}'
-        log.exception(msg)
-        raise CachitoError(msg)
-
-    if not rv.ok:
-        log.error(
-            'The worker failed to set the dependencies on request %d. The status was %d. '
-            'The text was:\n%s',
-            request_id, rv.status_code, rv.text,
-        )
-        raise CachitoError(f'Setting the dependencies on request {request_id} failed')
+    for index in range(0, len(deps), config.batch_size):
+        payload = {'dependencies': deps[index:index + config.batch_size]}
```

Comment:
In this case, PEP8 recommends no space between the addition sign: https://www.python.org/dev/peps/pep-0008/#pet-peeves